src/widget/form/chatform.cpp

```diff
@@ -185,8 +185,12 @@ void ChatForm::onTextEditChanged()
 
 void ChatForm::onAttachClicked()
 {
-    QStringList paths = QFileDialog::getOpenFileNames(this,
-                                                      tr("Send a file"));
+    QStringList paths = QFileDialog::getOpenFileNames ( this,
+                                                      tr("Send a file"),
+                                                      QDir::homePath(),
+                                                      0,
+                                                      0,
+                                                      QFileDialog::DontUseNativeDialog );
```

Comment:
Won't that cause regression on windows, osx and KDE5, since they use native dialogs?